mcos/observation_simulator.py

```diff
@@ -40,4 +40,20 @@ def simulate(self) -> (np.array, np.array):
         return x.mean(axis=0).reshape(-1, 1), np.cov(x, rowvar=False)
 
 
+class MuCovJackknifeObservationSimulator(AbstractObservationSimulator):
 
+    def __init__(self, mu: np.array, cov: np.array, n_observations: int):
+        self.mu = mu
+        self.cov = cov
+        self.n_observations = n_observations
+
+    def simulate(self) -> (np.array, np.array):
+        x = np.random.multivariate_normal(self.mu.flatten(), self.cov, size=self.n_observations)
+
+        x_prime = []
+
+        for i in range(len(x)):
+            x_prime.append((np.sum(x, axis=0) - x[i]))
+
+        return (np.asarray(x_prime) / (self.n_observations - 1)).mean(axis=0).reshape(-1, 1),\
```

Comment:
I have a hard time believing this is correct. `x_prime` is a sum of random variables. I wouldn't think the covariance of a sum would have the same magnitude of covariances as the original.